organizations.go

```diff
@@ -64,11 +65,12 @@ func (o OrganizationCreateRequest) String() string {
 
 // OrganizationUpdateRequest type used to update an Equinix Metal organization
 type OrganizationUpdateRequest struct {
-	Name        *string `json:"name,omitempty"`
-	Description *string `json:"description,omitempty"`
-	Website     *string `json:"website,omitempty"`
-	Twitter     *string `json:"twitter,omitempty"`
-	Logo        *string `json:"logo,omitempty"`
+	Name        *string  `json:"name,omitempty"`
+	Description *string  `json:"description,omitempty"`
+	Website     *string  `json:"website,omitempty"`
+	Twitter     *string  `json:"twitter,omitempty"`
+	Logo        *string  `json:"logo,omitempty"`
+	Address     *Address `json:"address,omitempty"`
```

Comment:
We could potentially send an AddressUpdateRequest object if the API supports updates to individual fields.

https://github.com/packethost/packngo/blob/38eb9a9d3ae9c27ee1bbd7bb720d3fa362873eca/facilities.go#L35-L45

I think it's ok to treat the whole address object as a single entity in this, as is done here.